Cargo.toml

```diff
@@ -259,8 +259,8 @@ ya-sb-util = { git = "https://github.com/golemfactory/ya-service-bus.git", rev =
 #ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "2a6350f62cf8d926721225a3451822731456e3fe" }
 
 ## RELAY and networking stack
-# ya-relay-stack = { git = "https://github.com/golemfactory/ya-relay.git", rev = "f9371b572204365920bbe1f6cc9b3e757bf12887" }
-# ya-relay-client = { git = "https://github.com/golemfactory/ya-relay.git", rev = "f9371b572204365920bbe1f6cc9b3e757bf12887" }
+ya-relay-stack = { git = "https://github.com/golemfactory/ya-relay.git", rev = "d8e2246d1a6bda7e2c7044aed785a0e7c5cf7316" }
```

Comment:
It is a commit from master replacing commit from merged branch https://github.com/golemfactory/ya-relay/pull/229/commits